app/livechat/client/stylesheets/livechat.less

```diff
@@ -5,6 +5,57 @@
 @secondary-font-color: #7f7f7f;
 @info-font-color: #aaaaaa;
 
+.lc-time {
+  float: left;
+  margin-top: -0px;
+  & > .lc-time-picker-btn {
+    font-weight: inherit;
+    font-size: inherit;
+    color: #383838;
+  }
+
+  & > .lc-time-picker-btn {
+    position: relative;
+
+    .fade {
+      color: #9ea2a8;
+    }
+  }
+}
+
+.map-container {
+  width: 500px;
+  max-width: 100%;
+  height: 500px;
+  overflow-x: scroll;
+}
+
+.lc-time-from,
+.lc-time-to {
+  font-weight: 400;
+  font-size: 15px;
+}
+
+.table-location {
+  font-family: arial, sans-serif;
+  border-collapse: collapse;
+  width: 100%;
+}
+
+td, th {
+  border: 1px solid #2e0707;
+  text-align: left;
+  padding: 8px;
+}
+
+.rc-table-tr {
+  height: 60px;
+}
+
+tr:nth-child(even) {
```

Comment:
If you want a different style for your table, you need to apply it to a specif class or something like that.